### PR TITLE
Cleanups around image metadata tarball and export

### PR DESCRIPTION
**Pull request by meatballhat**

### CODE REVIEW

**Comment by meatballhat** [comment]
Workin': https://travis-ci.org/travis-infrastructure/packer-build/jobs/193775423

``` bash
curl -sSL https://s3.amazonaws.com/travis-infrastructure-packer-build-artifacts/travis-ci/packer-templates/ci-connie/googlecompute/image-metadata-travis-ci-connie-trusty-1484926599.tar.bz2 | tar -tf -
```

```
image-metadata-travis-ci-connie-trusty-1484926599/
image-metadata-travis-ci-connie-trusty-1484926599/system_info.json
image-metadata-travis-ci-connie-trusty-1484926599/env/
image-metadata-travis-ci-connie-trusty-1484926599/env/TRAVIS_COOKBOOKS_BRANCH
image-metadata-travis-ci-connie-trusty-1484926599/env/TRAVIS_COOKBOOKS_SHA
image-metadata-travis-ci-connie-trusty-1484926599/env/PACKER_TEMPLATES_BRANCH
image-metadata-travis-ci-connie-trusty-1484926599/env/PACKER_BUILD_NAME
image-metadata-travis-ci-connie-trusty-1484926599/env/PACKER_BUILDER_TYPE
image-metadata-travis-ci-connie-trusty-1484926599/env/PACKER_TEMPLATES_SHA
image-metadata-travis-ci-connie-trusty-1484926599/env/TRAVIS_COOKBOOKS_DIR
image-metadata-travis-ci-connie-trusty-1484926599/travis_packer_templates_rspec.json
image-metadata-travis-ci-connie-trusty-1484926599/job-board-register.yml
image-metadata-travis-ci-connie-trusty-1484926599/node-attributes.yml
image-metadata-travis-ci-connie-trusty-1484926599/dpkg-manifest.json
```

File: bin/packer-build-script [hunk 1]
```diff
@@ -1,12 +1,63 @@
...
+  echo "     Writing ${packer_build_image_name} to:" >&2
+  echo "         ${top}/.packer-build-image-name" >&2
+  echo "${packer_build_image_name}" >"${top}/.packer-build-image-name"
+}
+
+__abspath() {
+  python <<EOPYTHON
+import os, sys
+sys.stdout.write(os.path.abspath("${1}") + "\\n")
+EOPYTHON
```
Comment (igorwwwwwwwwwwwwwwwwwwww) [inline bin/packer-build-script]: whoa

Comment (igorwwwwwwwwwwwwwwwwwwww) [inline bin/packer-build-script]: perhaps use argv inside of the python script to avoid escaping issues?

Comment (meatballhat) [inline bin/packer-build-script]: When life gives you GNU vs. BSD 🍋 , rub some Python on it 🐍 .

Comment (meatballhat) [inline bin/packer-build-script]: @igorwwwwwwwwwwwwwwwwwwww I'm +0 to using `sys.argv` in there.  Maybe a future iteration if there are issues?

**Pull request merged**